### ISSUE TITLE
Strange CallbackBehaviorAttribute exposed in system.servicemodel.duplex

### ISSUE THREAD

**Issue by thrixton** [dotnet/wcf#2772]
When applying a CallbackBehaviorAttribute to a callback implementation, there is no option to specify the ConcurrencyMode as documented at [docs.microsoft.com](https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.callbackbehaviorattribute?view=netframework-4.7.2&viewFallbackFrom=netstandard-2.0).
Looking through the code there appears to be a partial implementation [here](https://github.com/dotnet/wcf/blob/c8a7d554875fb65f1bee27e7721bc7d98b5bab4c/src/System.ServiceModel.Duplex/ref/System.ServiceModel.Duplex.cs) which does not have the ConcurrencyMode property, the full implementation [here](https://github.com/dotnet/wcf/blob/a2c95ad9e5cb2fff465655ae234454cf6f1c0f83/src/System.Private.ServiceModel/src/System/ServiceModel/CallbackBehaviorAttribute.cs) does, however the partial implementation appears to be exposed somehow.
I'm using version [4.4.0](https://www.nuget.org/packages/System.ServiceModel.Duplex/4.4.0) from nuGet and running visual studio 2017.

## Incorrect implementation
```
[AttributeUsage(AttributeTargets.Class)]
public sealed class CallbackBehaviorAttribute : Attribute, IEndpointBehavior
{
  public CallbackBehaviorAttribute();

  public bool AutomaticSessionShutdown { get; set; }
  public bool UseSynchronizationContext { get; set; }
}
```

**Comment by KKhurin** [comment]
Thanks @thrixton for the feedback. We've looked at this problem and logged #1959 when working on wcf sessions tests. It was not a trivial task to add full support for all concurrency modes so we postponed it till we get more people asking for this.
We'd like to know more about your scenario. Is this missing feature blocking you? While there are possible workarounds they would make the code unnecessarily complex and less elegant.
